src/components/SecurityUser/AddSecurityUserComponent.jsx

```diff
@@ -91,7 +97,7 @@ class AddSecurityUserComponent extends Component {
             required
           />
 
-          {emailError && (
+          {(!emailValid && email !== '') && (
```

Comment:
Is the second check i.e. `email !== ''` necessary? I think it's redundant because an empty empty string is already an invalid email so our email validation function should flag it as invalid

Reply:
I think it's necessary since we don't want the user to see an error message when they haven't yet even typed an email in the input. Without the second check, the form loads with the error message already displayed by default

Reply:
Oh ok. That's fine then

Reply:
Cool